crates/rpc/rpc/src/debug.rs

```diff
@@ -646,24 +645,14 @@ where
                                     );
 
                                 if let Some(account) = &account.account {
-                                    if include_preimages {
-                                        keys.insert(
-                                            hashed_address,
-                                            alloy_rlp::encode(address).into(),
-                                        );
-                                    }
+                                    keys.insert(hashed_address, alloy_rlp::encode(address).into());
 
                                     for (slot, value) in &account.storage {
                                         let slot = B256::from(*slot);
                                         let hashed_slot = keccak256(slot);
                                         storage.storage.insert(hashed_slot, *value);
 
-                                        if include_preimages {
-                                            keys.insert(
-                                                hashed_slot,
-                                                alloy_rlp::encode(slot).into(),
-                                            );
-                                        }
+                                        keys.insert(hashed_slot, alloy_rlp::encode(slot).into());
```

Comment:
also, we should rly remove rlp encoding here. makes no sense

Reply:
done